returnn/training.py

```diff
@@ -130,6 +132,7 @@ def __init__(
         time_rqmt: float = 4,
         mem_rqmt: float = 4,
         cpu_rqmt: int = 2,
+        launch_cmd: str = "mpirun",
```

Comment:
As this is only related to horovod, would `horovod_launch_command` not be the better name?

Or are there other purposes for this? Then the default should be `None`.

Reply:
`horovod_num_processes` is misleading, we just kept the name to not break setups. It means any kind of multi GPU training, also without Horovod, also PyTorch. So having `horovod_` prefix here does not make sense.
(Maybe we can rename `horovod_num_processes` in some non-breaking way, but I'm not sure, and this should be a separate PR.)

But you are right, a better name might be `distributed_launch_cmd` or sth like that.

The default cannot be `None` here because that would also break setups. E.g. for any setup which has `horovod_num_processes` not None, it should keep the current behavior, i.e. use `mpirun`. I think this default is fine, when the docs explain that it is only used if `horovod_num_processes` is not None.